examples/text-generation/run_generation.py

```diff
@@ -91,6 +91,12 @@ def setup_parser(parser):
         type=str,
         help="If `--dataset_name` was given, this will be the name of the column to use as prompts for generation.",
     )
+    parser.add_argument(
+        "--use_cache",
+        default=None,
+        type=bool,
+        help="",
```

Comment:
empty help

Reply:
Addressed, removed use_cache parameter as it is redundant.